doc/classes/@GlobalScope.xml

```diff
@@ -2411,6 +2411,8 @@
 		<constant name="PROPERTY_HINT_RANGE" value="1" enum="PropertyHint">
 			Hints that an integer or float property should be within a range specified via the hint string [code]"min,max"[/code] or [code]"min,max,step"[/code]. The hint string can optionally include [code]"or_greater"[/code] and/or [code]"or_lesser"[/code] to allow manual input going respectively above the max or below the min values. Example: [code]"-360,360,1,or_greater,or_lesser"[/code].
 		</constant>
+		<constant name="PROPERTY_HINT_ANGLE" value="7" enum="PropertyHint">
+		</constant>
```

Comment:
```suggestion
		<constant name="PROPERTY_HINT_ANGLE" value="7" enum="PropertyHint">
			Hints that a float property represents an angle in [i]radians[/i], but it will be edited as [i]degrees[/i] in the inspector. When using this property hint, value hints in the property hint string will also have to be specified as [i]degrees[/i].
		</constant>
```